packages/@lwc/template-compiler/src/__tests__/fixtures/html-tags/unkown-element-orphaned/actual.html

```diff
@@ -0,0 +1,5 @@
+<template>
+  <tr>
+    <unkonwn></unkonwn>
+  </tr>
+</template>
```

Comment:
Given that the algorithm relies on checking aunt/uncle nodes, it would be nice to have some tests with multiple aunt/uncle nodes to confirm it's working correctly.

Reply:
Thought of the same thing!